js/util/util.js

```diff
@@ -465,3 +465,12 @@ exports.arraysIntersect = function(a, b) {
     }
     return false;
 };
+
+var warnOnceHistory = {};
```

Comment:
This will log warnings once per page instead of once per map. Based on what we log right now this seems ok to me though.